README.md

```diff
@@ -1,2 +1,51 @@
 # LDK Node
-A ready-to-go node implementation built using LDK.
+A ready-to-go Lightning node library built using [LDK](https://lightningdevkit.org/) and [BDK](https://bitcoindevkit.org/).
+
+LDK Node is a non-custodial Lightning node in library form. Its central goal is to provide a small, simple, and straightforward interface that enables users to easily setup and run a Lightning node with an integrated on-chain wallet. While minimalism is at its core, LDK Node aims to be sufficiently modular and configurable to be useful for a variety of use cases. 
```

Comment:
@moneyball Any opinion on self-custodial vs non-custodial?

Reply:
I don't feel strongly about it. Non-custodial is fine.